templates/zombienet/tests/smoke.rs

```diff
@@ -7,28 +7,73 @@
 //! `cargo build --package minimal-template-node --release`
 //! `export PATH=<path-to-polkadot-sdk-repo>/target/release:$PATH
 //!
-//! The you can run the test with
-//! `cargo test -p template-zombienet-tests`
+//! There are also some tests related to omni node which run basaed on pre-generated chain specs,
+//! so to be able to run them you would need to generate the right chain spec (just minimal and
+//! parachain tests supported for now).
+//!
+//! You can run the following command to generate a minimal chainspec, once the runtime wasm file is
+//! compiled:
+//!`chain-spec-builder create --relay-chain <relay_chain_id> --para-id 1000 -r \
+//!     <path_to_template_wasm_file> named-preset development`
+//!
+//! Once the files are generated, you must export an environment variable called
+//! `RELAY_CHAIN_SPECS_DIR` which should point to the absolute path of the directory
```

Comment:
I would propose to change the name to `CHAIN_SPECS_DIR` as it may also contain parachain and minimal (which is not prachain and is not related to relaychain).  `RELAY_CHAIN_SPECS_DIR` sounds a bit confusing for me.

Reply:
Resolved here: [3486884](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/348688497cf97a22119e4704221a2c4edf851a98)